src/integrations/feed/clients.ts

```diff
@@ -0,0 +1,167 @@
+import { FeedConfig, FeedResponse, IFeedClient } from './types';
+import fetch, { RequestInit } from 'node-fetch';
+import { fetchOptions as globalFetchOptions } from '../../http';
+import pRetry, { AbortError } from 'p-retry';
+import { IORedisPool } from '@dailydotdev/ts-ioredis-pool';
+import { Context } from '../../Context';
+
+const ONE_DAY_SECONDS = 24 * 60 * 60;
```

Comment:
We already have this in `src/redis` maybe make it a common util export instead?

Reply:
Good point!